terraform/aws/eks-ebs-csi-addon/eks-cluster.tf

```diff
@@ -0,0 +1,38 @@
+module "eks" {
+  source  = "terraform-aws-modules/eks/aws"
+  version = "19.5.1"
+
+  cluster_name    = local.cluster_name
+  cluster_version = var.cluster_version
+
+  vpc_id     = module.vpc.vpc_id
+  subnet_ids = module.vpc.private_subnets
+  cluster_endpoint_public_access = true
+
+  eks_managed_node_group_defaults = {
+    ami_type = "AL2_x86_64"
+
+  }
+
+  eks_managed_node_groups = {
```

Comment:
[disk_size](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/eks_node_group#disk_size) should be specified with the default value of 80 GB, because Kubecost installation with persistent storage requires 64 GB as minimum.

Reply:
Yes, I can add this in and will do another test run and then push my changes

Reply:
Actually, the disk size in this scenario is the disk attached directly to the compute node, not the PV disks requested by kubecost.  The PVs get requested with PVCs when cost-analyzer is coming up, so we do not need a large base disk attached to the computer node.  I tested the basic helm install of kubecost once the infra came up with smaller computer disks and it came up just fine.

Reply:
Yeah, you are right ! it was my assumption from an issue I encountered in the early EKS version few years ago (IAM permission issue).